_posts/2021-09-27-changelog-156.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #156"
+date: 2021-09-27 13:00:00 +0300
+---
+
+This is the last release that supports the 2021.1 IDEs. To receive plugin updates further, please upgrade
+your IDE to 2021.2.
+
+## New Features
+
+* [#7317] Enable WSL support by default
```

Comment:
Let's provide more detailed text about what works and doesn't work now and where. For example, debugging works only in CLion and you need special toolchain for it. Profiling doesn't work at all for now